### PR TITLE
Closes #2309 Fix favicon

### LINKED ISSUE #2309: [BUG] Favicon is not showing
### Description

The favicon is not showing on the main site, kindly use these icons:

[favicon_io.zip](https://github.com/rupali-codes/LinksHub/files/14633308/favicon_io.zip)

### Screenshots

image

### Any additional information?

_No response_

### What browser are you seeing the problem on?

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by bottlewook**
## Fixes Issue
Closes #2309

## Changes proposed
Reflecting favicons

## Screenshots
스크린샷 2024-03-18 오후 10 35 06

### CODE REVIEW

**Review by rupali-codes — DISMISSED** [review]
lgtm

File: public/favicon/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (aftabrehan) [inline public/favicon/site.webmanifest:1]:
```suggestion
{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#714eff","background_color":"#714eff","display":"standalone"}
```

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @bottlewook 💪 🚀 💯 

It just needs minor adjustment, and then we're good to go 🚀

**Review by aftabrehan — APPROVED** [review]

**Comment by aftabrehan** [comment]
@Anmol-Baranwal, @CBID2 Please, help to review and merge the PR.

**Comment by bottlewook** [comment]
Invite me back next time! I'm having so much fun contributing.

**Comment by bottlewook** [comment]
@rupali-codes @aftabrehan Invite me back next time! I'm having so much fun contributing.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]

**Comment by rupali-codes** [comment]
> @rupali-codes @aftabrehan Invite me back next time! I'm having so much fun contributing.

Sure!!, Please keep an eye on the open issues with [up for grabs](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22) label

**Pull request merged**